docs/src/main/sphinx/release/release-449.md

```diff
@@ -35,9 +35,10 @@
 ## Hive connector
 
 * Add support for specifying a catalog name in the Thrift metastore with the
-  `hive.metastore.thrift.catalog-name` configuration property. (`10287`)
-* Add support for configuring the query retry policy on the S3 filesystem with
-  the `s3.retry-mode` and `s3.max-error-retries` configuration properties.
+  `hive.metastore.thrift.catalog-name` configuration property. ({issue}`10287`)
+* Add support for configuring the query retry policy on the native S3 filesystem
```

Comment:
I would suggest to use "request retry policy" instead of "query retry policy", so it is more obvious that we are no talking about Trino queries.

Reply:
Done here and all others.